### DISCUSSION TITLE
For the code editor, line_spacing should allow negative values

### DISCUSSION THREAD

**Discussion by jokoon** [godotengine/godot-proposals#9787] [Editor]
Currently, I cannot set negative values for line_spacing in the settings.

For example, sublime text allows those settings:

```
"line_padding_bottom": -1,
"line_padding_top": -1,
```

Personally, I like having a more compact look, and negative values allow it.

To be clear: I am talking about the godot editor, not UI elements for games I make with godot.

Maybe there's a hacky wait to change that parameter, bypassing the setting panel? EDIT: I changed the value in the config file, did not help.

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot/issues/76385.
